packages/plugins/input-exercise/__tests__/index.tsx

```diff
@@ -0,0 +1,229 @@
+import { EditorPlugin } from '@edtr-io/plugin'
+import { plugin as inputExercisePlugin } from '@edtr-io/plugin-input-exercise/__fixtures__'
+import { plugin as textPlugin } from '@edtr-io/plugin-text/__fixtures__'
+import { Renderer } from '@edtr-io/renderer'
+import { render } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import React from 'react'
+
+describe('input challenge', () => {
```

Comment:
You don't need a single top-level `describe` block since jest already groups tests by files.